cypress/e2e/tests/migration/applicationinventory/manageimports/pagination.test.ts

```diff
@@ -27,10 +27,11 @@ import {
     deleteAppImportsTableRows,
 } from "../../../../../utils/utils";
 import { trTag } from "../../../../types/constants";
-import { actionButton } from "../../../../views/applicationinventory.view";
+import { actionButton, sideKebabMenuImports } from "../../../../views/applicationinventory.view";
 
 import * as commonView from "../../../../views/common.view";
 import { Application } from "../../../../models/migration/applicationinventory/application";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
```

Comment:
It doesn't look this is needed in this file .

Reply:
Solved in 90bcc68e960ea58a89cbfac10533e3a1c6966a55.